test/action.yaml

```diff
@@ -7,12 +7,45 @@ inputs:
     default: true
   github-token:
     required: true
-    description: "GitHub token passed to coveralls. Usually - 'secrets.GITHUB_TOKEN'. Coveralls uses this token to verify the posted coverage data on the repo and create a new check based on the results. It is built into Github Actions and does not need to be manually specified in your secrets store."
+    description: "GitHub Personal Access Token that can checkout the consumer git repository. e.g. 'secrets.GITHUB_TOKEN'"
+  tf-cli-credentials-token:
+    required: true
+    description: "The terraform cli config credentials token"
+  tf-cli-config-file:
+    required: false
+    description: "Relative or absolute path to the terraform cli config file"
+    default: ./.terraformrc
+
 runs:
   using: composite
   steps:
     - uses: actions/checkout@v2
       if: inputs.checkout-repo
-    - run: |
-        echo "Test not implemented yet"
+      with:
+        fetch-depth: 0
+    - uses: hashicorp/setup-terraform@v1
+      with:
+        terraform_version: 1.1.2
+        cli_config_credentials_token: ${{ inputs.tf-cli-credentials-token }}
+    - name: verify test setup file exists
+      if: hashFiles('./script/setup') == ''
+      shell: bash
+      run: |
+        echo ./script/setup file is required but missing
+        exit 1
+    - name: verify test file exists
+      if: hashFiles('./script/test') == ''
+      shell: bash
+      run: |
+        echo ./script/test file is required but missing
```

Comment:
We probably don't explicitly need to check for this, though I guess it makes the error nicer.  the run steps below should just fail.